tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: install epel-release
+  yum:
+    name: epel-release
```

Comment:
This package doesn't exists in RHEL repositories, only CentOS. The whole ideia of this role is to address this issue, otherwise I would just install this package everywhere and I would solve the problem with just one task.

Furthermore, even in CentOS, you are doing the installation twice when you replace a file provided by the package manually. This makes no sense.